Wikimate.php

```diff
@@ -1406,6 +1404,29 @@ public function upload($data, $comment, $text = null, $overwrite = false)
 	}
 
 	/**
+	 * Uploads the given contents to the current file.
+	 * $text is only used for the article page of a new file, not an existing
```

Comment:
I don't think a file's page is called "article page". I'd use just "page", or maybe "page contents".

Also, I'd split the line in the comma, since that helps the reading flow and conforms to [semantic linebreaking](http://rhodesmill.org/brandon/2012/one-sentence-per-line/).

Reply:
For future reference, #135 introduces the semantic line breaks recommendation explicitly into the project docs.